lib/utils/forkedstoragetrie.js

```diff
@@ -1,41 +1,26 @@
-var MerklePatriciaTree = require("merkle-patricia-tree");
+const MerklePatriciaTree = require("merkle-patricia-tree");
+const BaseTrie = require("merkle-patricia-tree/baseTrie");
+const checkpointInterface = require("merkle-patricia-tree/checkpoint-interface");
 var utils = require("ethereumjs-util");
 var inherits = require("util").inherits;
 var Web3 = require("web3");
 var to = require("./to.js");
 
-inherits(ForkedStorageTrie, MerklePatriciaTree);
+inherits(ForkedStorageBaseTrie, BaseTrie);
```

Comment:
Reason for change:

Most of these changes are just code reorg. We now have a "Base" trie and our normal trie. This is just to align with the MPT implementation. I wanted to use the `copy` mechanism, but couldn't do so without this small refactor... because of reasons (things crashed).